src/Simulation/QCTraceSimulator.Tests/Circuits/MetricsCalculationTests.qs

```diff
@@ -26,7 +26,35 @@ namespace Microsoft.Quantum.Simulation.QCTraceSimulatorRuntime.Tests {
         }
     }
     
-    
+    operation TDepthWidth () : Unit {
+        // q2 q1 q3 q4
+        // |  |      
+        // O--O      
+        // |  |      
+        //    |  |  |
+        //    |  O--O
+        //    |  |  |
+        //    |  |   
+        //    O--O   
+        //    |  |   
+        //           
+        // Width 4, Depth 2.
+        // Alternatively q2 is reused for q4 (or q3)
+        // Width 3, Depth 3.
+
+        using (q1 = Qubit()) {
+            using (q2 = Qubit()) {
+                Controlled Z([q2], q1);
```

Comment:
Controlled Z is a Clifford operation; why is a non-zero 𝑇-depth reported for a sequence of Clifford operations?

Reply:
Good question! The new tracer is configurable. For this test it is specifically configured to treat CZ as depth 1. I'll add a comment about this.

Reply:
Updated comment.